components/in_memory_engine/src/region_stats.rs

```diff
@@ -184,32 +191,36 @@ impl RegionStatsManager {
             for region in &curr_top_regions {
                 let _ = region_loaded_map.insert(region.0.id, Instant::now());
                 if !cached_regions.contains_key(&region.0.id) {
+                    IN_MEMORY_ENGINE_AUTO_LOAD_EVICT_TOP_REGION_COP_REQ
+                        .observe(region.1.query_stats.coprocessor as f64);
+                    IN_MEMORY_ENGINE_AUTO_LOAD_EVICT_TOP_REGION_MVCC_AMP
+                        .observe(region.1.cop_detail.mvcc_amplification());
                     regions_to_load.push(region.0.clone());
                 }
             }
             regions_to_load
         };
 
-        {
-            // TODO(SpadeA): remove it after it's stable
-            let debug: Vec<_> = cached_region_stats
-                .iter()
-                .map(|crs| {
-                    format!(
-                        "region_id={}, cop={}, avg_cop={}, cop_detail={:?}, mvcc_amplification={}",
-                        crs.region.id,
-                        crs.stat.query_stats.coprocessor,
-                        crs.sma_cop_requests_avg,
-                        crs.stat.cop_detail,
-                        crs.stat.cop_detail.mvcc_amplification(),
-                    )
-                })
-                .collect();
-            info!(
-                "ime collect regions activities";
-                "regions" => ?debug,
-            );
+        let mut debug = Vec::with_capacity(cached_region_stats.len());
+        for crs in &cached_region_stats {
+            debug.push(format!(
+                "region_id={}, cop={}, avg_cop={}, cop_detail={:?}, mvcc_amplification={}",
+                crs.region.id,
+                crs.stat.query_stats.coprocessor,
+                crs.sma_cop_requests_avg,
+                crs.stat.cop_detail,
+                crs.stat.cop_detail.mvcc_amplification(),
+            ));
+            IN_MEMORY_ENGINE_AUTO_LOAD_EVICT_CACHED_REGION_COP_REQ
+                .observe(crs.stat.query_stats.coprocessor as f64);
+            IN_MEMORY_ENGINE_AUTO_LOAD_EVICT_CACHED_REGION_MVCC_AMP
+                .observe(crs.stat.cop_detail.mvcc_amplification());
         }
+        // TODO(SpadeA): remove it after it's stable
```

Comment:
Do we still need it ?

Reply:
Yes, it's kept for debugging auto load evict.